### PR TITLE
app-admin/doctl: new ebuild

### PR DESCRIPTION
**Pull request by SpiderX**
A command line tool for DigitalOcean services

Signed-off-by: Vladimir Pavljuchenkov <spiderx@spiderx.dp.ua>
Package-Manager: Portage-2.3.62, Repoman-2.3.11

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
Hi there,

Sometimes 'new package' PRs get snowed under by all the other ones (new packages are low priority compared to fixes to already existing packages). If you'd like to make this ebuild available to the Gentoo users anyway, might I suggest the [GURU overlay](https://wiki.gentoo.org/wiki/Project:GURU). This is an ebuild repository maintained collaboratively by Gentoo users. To gain access to the repository, simply create an "Access request" on bugs.gentoo.org.

**Comment by SpiderX** [comment]
@AndrewAmmerlaan Thanks, but I already have my own overlay and supporting another one without obvious benefits for me would be a waste of efforts from my point of view. Anyway, good luck with GURU.

File: app-admin/doctl/doctl-1.39.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+DESCRIPTION="A command line tool for DigitalOcean services"
+HOMEPAGE="https://github.com/digitalocean/doctl"
+SRC_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+BDEPEND=">=dev-lang/go-1.13"
```
Comment (juippis) [inline app-admin/doctl/doctl-1.39.0.ebuild:19]: Not needed unless the latest version specifies a more recent requirement.

**Review by juippis — COMMENTED** [review]
Could you update this to latest version?

Also FEATURES="test" failed for me. Coul you take a look, and restrict it if it's impossible to fix.

File: app-admin/doctl/doctl-1.39.0.ebuild (hunk 1 above)
Comment (SpiderX) [reply app-admin/doctl/doctl-1.39.0.ebuild]: There were a need in this, but no more. Removed.

**Comment by SpiderX** [comment]
Updated to the latest version. Test phase fixed.

File: app-admin/doctl/doctl-1.54.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+EAPI=7
+
+inherit bash-completion-r1 go-module
+
+DESCRIPTION="A command line tool for DigitalOcean services"
+HOMEPAGE="https://github.com/digitalocean/doctl"
+SRC_URI="https://github.com/digitalocean/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
```
Comment (juippis) [inline app-admin/doctl/doctl-1.54.1.ebuild:12]: Sorry that I failed to mention this the first time, but these packages need to include licenses for bundled go modules as well. There are tools in the ::gentoo repo to check these, but I can't say which is the best one. See go-licenses, golicense and go-license-detector.

Comment (SpiderX) [reply app-admin/doctl/doctl-1.54.1.ebuild]: I know about these tools and use golicense in new ebuilds, this one is relatively old and I didn't run golicense against it.
But what I didn't know that mentioning all licenses in ebuild is mandatory. Will take this into account, thanks.
License for go modules added.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**